<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Export</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0,0">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Export your complete collection.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MessageLabel" name="message">
     <property name="text">
      <string>Message</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Theme</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboTheme"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Items to export</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chkMovies">
       <property name="text">
        <string>Movies</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="chkTvShows">
       <property name="text">
        <string>TV Shows</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="chkConcerts">
       <property name="text">
        <string>Concerts</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QProgressBar" name="progressBar">
     <property name="value">
      <number>24</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="text">
        <string>Close</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnExport">
       <property name="text">
        <string>Export</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageLabel</class>
   <extends>QLabel</extends>
   <header>ui/small_widgets/MessageLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>btnExport</sender>
   <signal>clicked()</signal>
   <receiver>ExportDialog</receiver>
   <slot>onBtnExport()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboTheme</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>ExportDialog</receiver>
   <slot>onThemeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>71</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>ExportDialog</receiver>
   <slot>onBtnClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnExport()</slot>
  <slot>onThemeChanged()</slot>
  <slot>onBtnClose()</slot>
 </slots>
</ui>
